staging/src/k8s.io/kubectl/pkg/polymorphichelpers/rollback_test.go

```diff
@@ -62,7 +62,7 @@ func TestGetDeploymentPatch(t *testing.T) {
 		t.Errorf("expected strategic merge patch, got %v", patchType)
 	}
 	expectedPatch := `[` +
-		`{"op":"replace","path":"/spec/template","value":{"metadata":{"creationTimestamp":null},"spec":{"containers":[{"name":"","image":"foo","resources":{}}]}}},` +
+		`{"op":"replace","path":"/spec/template","value":{"metadata":{"creationTimestamp":null},"spec":{"containers":[{"name":"","image":"foo","resources":{}}],"schedulerName":""}}},` +
```

Comment:
I think I didn't expect to see this show up in patches, maybe it makes sense though :/

Reply:
It looks like this patch is from empty PodSpec. So yeah, it makes sense.